contrib/haproxy/files/haproxy.cfg

```diff
@@ -0,0 +1,43 @@
+#---------------------------------------------------------------------
+# Simple configuration for an HTTP proxy listening on port 80 on all
+# interfaces and forwarding requests to a single backend "servers" with a
+# single server "server1" listening on 127.0.0.1:8000
+# 
+# for more visit:
+#   https://docs.haproxy.org/2.9/configuration.html
+#
+#---------------------------------------------------------------------
+
+global
+    daemon
+    maxconn 256
+
+defaults
+    mode http
+    timeout connect 5000ms
+    timeout client 50000ms
+    timeout server 50000ms
+
+frontend http-in
+    bind *:80
+    default_backend servers
+
+backend servers
+    server server1 127.0.0.1:8000 maxconn 32
+
+
+# The same configuration defined with a single listen block. Shorter but
+# less expressive, especially in HTTP mode.
+global
+    daemon
+    maxconn 256
+
+defaults
+    mode http
+    timeout connect 5000ms
+    timeout client 50000ms
+    timeout server 50000ms
+
+listen http-in
+    bind *:80
+    server server1 127.0.0.1:8000 maxconn 32
```

Comment:
this is the same config twice though, why?

Reply:
I took straight from the docuemntation, see https://docs.haproxy.org/2.9/configuration.html#2.7

Reply:
```
Assuming haproxy is in $PATH, test these configurations in a shell with:

    $ sudo haproxy -f configuration.conf -c
```


```
/ # haproxy -c -f x.cfg 
[NOTICE]   (12) : haproxy version is 2.8.5-aaba8d0
[ALERT]    (12) : config : parsing [x.cfg:23] : 'maxconn' already specified. Continuing.
[ALERT]    (12) : config : Parsing [x.cfg:31]: proxy 'http-in' has the same name as frontend 'http-in' declared at x.cfg:11.
[ALERT]    (12) : config : Error(s) found in configuration file : x.cfg
[ALERT]    (12) : config : Fatal errors found in configuration.
```